src/main/resources/public/src/utils/plugins/i18n.js

```diff
@@ -0,0 +1,97 @@
+import Vue from 'vue'
+import VueI18n from 'vue-i18n'
+
+Vue.use(VueI18n)
+
+export const i18n = new VueI18n({
+    locale: 'fr',
+    fallbackLocale: 'fr',
+    messages: {
+        en: {
+            dashboard: 'Dashboard',
+            data_request: 'Data request',
+            connectors_management: 'Connectors management',
+            statistics: 'Statistics',
+            dc_model: 'DC Model',
+            date: 'Date',
+            organization: 'Organization | Organizations',
+            state: 'State',
+            request_access_to_dataset: 'Request for access to dataset',
+            search_organization: 'Search a organization',
+            email: 'Email',
+            model: 'No model | Model | Models',
+            search_model: 'Search a model',
+            save: 'Save',
+            search_by_organization: 'Search by organization',
+            application_name: 'Application Name',
+            search_by_application: 'Search by application',
+            organization_name: 'Organization Name',
+            siret_number: 'SIRET number',
+            data_access_check: 'Data access check',
+            authorize_access_to_all_checked_data: 'Authorize access to all checked data of "{modelName}" model',
+            field: 'Field | Fields',
+            validate: 'Validate',
+            refuse: 'Refuse',
+            processing_statistics: 'Processing Statistics',
+            since: 'Since',
+            processing_completed: 'Processing completed',
+            processed_model: 'Processed model | Processed models',
+            concerned_organization: 'Concerned organization | Concerned organizations',
+            processing_since: 'Processing since | Processings since',
+            search: 'Search',
+            by_model: 'By Model',
+            being_processed: 'being processed',
+            creation: 'Creation | Creations',
+            deletion: 'Deletion | Deletions',
+            update: 'Update | Updates',
+            by_organization: 'By Organization',
+            models_processed_for_selected_organization: 'Models processed for selected organization',
+            all: 'All',
+            for_organization: 'for Organization',
+            created_ago: 'Created ago'
+        },
+        fr: {
+            dashboard: 'Tableau de bord',
+            data_request: 'Demande de données',
```

Comment:
"Demande d'accès aux données" ?
"Accès aux données" ?

Reply:
> "Demande d'accès aux données" ?
Too long
> "Accès aux données" ?
Not appropriete for me

just "Demande" ?

Reply:
"Demande d'accès" ?